FIX: set sites/default directory to 755

dan2k3k4 opened the pull request.

Schnitzel commented:
I think we should do the permission fix just after `COPY . /app`, as then it will also appear in the `php` image, or as alternative we need to do this also in the `php` image, but it's probably better to do it only once.

dan2k3k4 commented:
updated @Schnitzel, moved it to be before the php container lines

I'm not sure if a Dockerfile with "empty spaces" does anything or causes new layers (I assume each line causes a new layer)...

Schnitzel commented:
looks good!

> I'm not sure if a Dockerfile with "empty spaces" does anything or causes new layers (I assume each line causes a new layer)...

in my understanding empty lines doesn't do anything, just if you run an actual command like `RUN` it will generate a new layer
